Misc/NEWS.d/next/Core_and_Builtins/2025-04-07-12-50-10.gh-issue-132176.PRDbkA.rst

```diff
@@ -0,0 +1,2 @@
+Fix a crash when :func:`type` receives a :class:`tuple` subclass as the
```

Comment:
This change doesn't fix a crash but an assertion error which only occurs on a Python debug build.